### ISSUE TITLE
Version 4.0.0 not saving CSS file

### ISSUE THREAD

**Issue by nwpappas** [vuejs/rollup-plugin-vue#183]
### Expected behavior

Sending `css` property should export CSS file.

```
    vue({
      css: './build/assets/css/bundle.css'
    }),
```

### Actual behavior

No file is exported.

### Steps to reproduce the behavior

Upgraded 'rollup-plugin-vue' from 2.5 to 4.0.0, with no changes to 'rollup.config.js'. The CSS files is no longer exported.

Reverting the plugin back to 2.5 the problem is solved.

**Comment by znck** [comment]
Yes. That part was intentionally left out. It in-lines CSS and injects at runtime.

**Comment by nwpappas** [comment]
Is the built-in export being added back, or will it be left out in favor of letting other plugins deal with it - such as https://github.com/thgh/rollup-plugin-css-only?

**Comment by znck** [comment]
It is left out for other plugins. We are trying to have a minimal API surface.
